lib/fluent/plugin/in_tail.rb

```diff
@@ -113,6 +113,8 @@ def initialize
     config_param :path_timezone, :string, default: nil
     desc 'Follow inodes instead of following file names. Guarantees more stable delivery and allows to use * in path pattern with rotating files'
     config_param :follow_inodes, :bool, default: false
+    desc 'Skip long lines which size is longer than specified length'
+    config_param :skip_long_lines, :size, default: nil
```

Comment:
This is unnatural parameter name.
_maximum_line_capacity_ or something which represents acceptable line capacity is better.

Reply:
@cosmo0920
The parameter which accepts `:size` use `_limit_size`, then how about `maximum_line_limit_size` ?

Reply:
It makes clear that it is different from `read_lines_limit`.

Reply:
Use a more meaningful option name.

Reply:
Yes. You can use what you feel better. :+1: